packages/next/build/webpack/plugins/wellknown-errors-plugin/index.ts

```diff
@@ -1,8 +1,9 @@
-import { Compiler } from 'webpack'
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Is this eslint disable needed? I don't see it in other files.

Reply:
Good point, yeah that was leftover from a previous iteration.